scripts/helpers/moduleNames.js

```diff
@@ -4,5 +4,6 @@ module.exports = [
   "audiences",
   "eventmerge",
   "libraryinfo",
-  "machinelearning"
+  "machinelearning",
+  "identity"
```

Comment:
`identity` is not excludable. I think renaming this file will help understand what goes in it. `libraryInfo` is not either.